### PR TITLE
Update default max_tree_depth to 2

### PR DESCRIPTION
**Pull request by gregorybchris**

### CODE REVIEW

**Review by imatiach-msft — APPROVED** [review]

File: responsibleai/responsibleai/_managers/causal_manager.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class CausalConstants:
     DEFAULT_MAX_CAT_EXPANSION = 50
     DEFAULT_TREATMENT_COST = 0
     DEFAULT_MIN_TREE_LEAF_SAMPLES = 2
-    DEFAULT_MAX_TREE_DEPTH = 3
+    DEFAULT_MAX_TREE_DEPTH = 2
```
Comment (gaugup) [inline responsibleai/responsibleai/_managers/causal_manager.py:31]: Is there a test checking this change?

Comment (gregorybchris) [reply responsibleai/responsibleai/_managers/causal_manager.py:31]: How would you want this to be tested?

Comment (gaugup) [reply responsibleai/responsibleai/_managers/causal_manager.py:31]: Maybe check the ploicy tree dimensions, somehow. I am not an expert on the causal. But it is strange that there is no test covering this scenario,

Comment (gregorybchris) [reply responsibleai/responsibleai/_managers/causal_manager.py:31]: Good point. I think I'll just add these tests to my other causal PR to speed things up. Waiting for gates to pass is taking too long

**Pull request closed without merging**